web/web_support/src/main/java/com/intuit/tank/util/RestSecurityFilter.java

```diff
@@ -36,6 +37,7 @@
 import com.intuit.tank.project.User;
 import com.intuit.tank.vm.settings.TankConfig;
 
+@WebFilter(urlPatterns = "/rest/*")
```

Comment:
@kevin-mcgoldrick Seems like `LoginFilter` & `AdminFilter` filtered via url-pattern ([example](https://github.com/intuit/Tank/blob/e24cabf35c088408d616f267d95003c713fda46a/web/web_ui/src/main/webapp/WEB-INF/web.xml#L89)) in the web.xml so those look good, but `RestSecurityFilter` is declared to explicityly filter on servlet-name: **Intuit Tank REST API**, so wondering if our annotation should utilize servletNames instead of urlPatterns for RestSecurityFilter? Otherwise my assumption that the Intuit Tank REST API servlet declared [here](https://github.com/intuit/Tank/blob/e24cabf35c088408d616f267d95003c713fda46a/web/web_ui/src/main/webapp/WEB-INF/web.xml#L119) will filter on that url pattern so it doesn't matter, but just checking here

Reply:
Resolved in conversation in-person